tests/cairo_native.rs

```diff
@@ -106,6 +106,7 @@ fn get_block_hash_test_failure2() {
 }
 
 #[test]
+#[ignore = "Broken until GetExecutionInfoV2 libfunc is implemented"]
```

Comment:
@fmoletta Just one question, why where this tests passing before and not now?
I don't think it is good idea to comment the erc20 tests

Reply:
Its due to updating the compiler to version 2.4.2, the tests all started failing due to the get_execution_info_v2 libfunc not being implemented

Reply:
Update: They are now passing after the cairo native version was updated, Ill remove the ignores

Reply:
Update: That was a lie